wallet/src/de/schildbach/wallet/WalletApplication.java

```diff
@@ -946,7 +965,8 @@ public void finalizeWipe() {
         notifyWalletWipe();
         PinRetryController.getInstance().clearPinFailPrefs();
         MnemonicCodeExt.clearWordlistPath(this);
-        TransactionsAdapter.resetPreferences(this);
+        // TODO: get rid of a separate file for this pref
+        getSharedPreferences(HistoryHeaderAdapter.PREFS_FILE_NAME, MODE_PRIVATE).edit().clear().apply();
```

Comment:
Many classes in DashPay have a separate file for their preferences which contains 1-2 items. This is a bit weird - it's better to group preferences by features or just have them all in the Configuration - we can split it later using datastore.